frontend/src/__tests__/cypress/cypress/e2e/projects/ProjectList.cy.ts

```diff
@@ -0,0 +1,201 @@
+        pathname: '/api/k8s/apis/project.openshift.io/v1/projects',
+      },
+      mockK8sResourceList([mockProjectK8sResource({})]),
+    ).as('createdProjects');
+
+    createProjectModal.findSubmitButton().click();
+
+    cy.wait('@createProjectRequest').then((interception) => {
+      expect(interception.request.body).to.eql({
+        apiVersion: 'project.openshift.io/v1',
+        kind: 'ProjectRequest',
+        metadata: {
+          name: 'test-project',
+        },
+        description: 'Test project description.',
+        displayName: 'My Test Project',
+      });
+    });
+    cy.wait('@createdProjects');
+    cy.url().should('include', '/projects/test-project');
+  });
+
+  it('should list the new project', () => {
+    initIntercepts();
+    cy.intercept(
+      {
+        method: 'GET',
+        pathname: '/api/k8s/apis/project.openshift.io/v1/projects',
+      },
+      mockK8sResourceList([mockProjectK8sResource({})]),
+    );
+    projectListPage.visit();
+    projectListPage.navigate();
+    projectListPage.findProjectScopeDropdown().click();
+    projectListPage.findDatascienceProjectOption('Data science projects').should('be.visible');
+    projectListPage
+      .findDatascienceProjectOption(
+        'Only projects created in the Data Science Dashboard are displayed.',
+      )
+      .should('be.visible');
+    projectListPage.findAllprojectOption('All available projects').should('be.visible');
+    projectListPage
+      .findAllprojectOption(
+        'All available projects created in the Data Science Dashboard and OpenShift are displayed.',
+      )
+      .should('be.visible');
+    projectListPage.findProjectScopeDropdown().click();
+    projectListPage.findCreateProjectButton().should('exist');
+    projectListPage.findJupyterLauncher().should('exist');
+    projectListPage.shouldHaveProjects();
+    projectListPage.findProjectFilterDropdown().findDropdownItem('Name').should('exist');
+    projectListPage.findProjectFilterDropdown().click();
+    projectListPage.findProjectFilterDropdown().findDropdownItem('User').should('exist');
+    projectListPage.findSearchBox().should('exist');
+    projectListPage.findProjectRow('Test Project').should('exist');
+    projectListPage.shouldHaveDSLabel();
+    //checking project table headers
+    projectListPage.findHeaderLength().should('have.length', 4);
+    projectListPage.findHeader('Name').should('exist');
+    projectListPage.findHeader('Workbench').should('exist');
+    projectListPage.findHeader('Status').should('exist');
+    projectListPage.findHeader('Created').should('exist');
+    projectListPage.findProjectRow('Test Project').findKebabAction('Edit project').should('exist');
+    projectListPage
+      .findProjectRow('Test Project')
+      .findKebabAction('Edit permissions')
+      .should('exist');
+    projectListPage
+      .findProjectRow('Test Project')
+      .findKebabAction('Delete project')
+      .should('exist');
+    cy.testA11y();
```

Comment:
Unnecessary here. We already assert upon page load.
We will also be adding a check to perform this assertion at the end of each test.